.github/workflows/build_and_test.yml

```diff
@@ -26,3 +24,27 @@ jobs:
       run: ./gradlew jar
     - name: Test
       run: ./gradlew test
+      
+  build-and-push-anchor-platform:
+    runs-on: ubuntu-latest
+    name: Push stellar/anchor-platform:sha to DockerHub
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Get image tag (short sha)
+      shell: bash
+      id: get_tag
+      run: echo ::set-output name=TAG::$(git rev-parse --short ${{ github.sha }} )
+
+    - name: Login to DockerHub
+      uses: docker/login-action@bb984efc561711aaa26e433c32c3521176eae55b
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
Where are these secrets stored?

Reply:
In GitHub, they're something Ops will configure.